### DISCUSSION TITLE
Epiverse-TRACE developer space - System Dependencies in R Packages & Automatic Testing

### DISCUSSION THREAD

**Discussion by giscus** [epiverse-trace/epiverse-trace.github.io#120] [Blog post discussion]
# Epiverse-TRACE developer space - System Dependencies in R Packages & Automatic Testing

A place for Epiverse-TRACE developers to share their reflections, learnings, and showcase their work.

https://epiverse-trace.github.io/posts/system-dependencies/

**Comment by Bisaloo** [comment]
If you'd prefer to explore this content in podcast format, you can listen to the chapter from @rpodcast on this post: https://podverse.fm/clip/k8RPsYA1Ee :radio:
